website/docs/syncing_metastore.md

```diff
@@ -33,17 +40,16 @@ Among them, following are the required arguments:
 ```
 Corresponding datasource options for the most commonly used hive sync configs are as follows:
 
-| HiveSyncConfig | DataSourceWriteOption | Description |
-| -----------   | ----------- | ----------- |
-| --database       | hoodie.datasource.hive_sync.database       | name of the target database in Hive       |
-| --table   | hoodie.datasource.hive_sync.table        | name of the target table in Hive        |
-| --user   | hoodie.datasource.hive_sync.username        | username for hive metastore        | 
-| --pass   | hoodie.datasource.hive_sync.password        | password for hive metastore        | 
-| --use-jdbc   | hoodie.datasource.hive_sync.use_jdbc        | use JDBC to connect to metastore        | 
-| --jdbc-url   | hoodie.datasource.hive_sync.jdbcurl        | Hive metastore url        |
-| --sync-mode   | hoodie.datasource.hive_sync.mode        | Mode to choose for Hive ops. Valid values are hms, jdbc and hiveql.        |
-| --partitioned-by   | hoodie.datasource.hive_sync.partition_fields        | Comma-separated column names in the table to use for determining hive partition.        |
-| --partition-value-extractor   | hoodie.datasource.hive_sync.partition_extractor_class        | Class which implements PartitionValueExtractor to extract the partition values. `SlashEncodedDayPartitionValueExtractor` by default.        |
+| HiveSyncConfig | DataSourceWriteOption | Default Value | Description |
+| -----------   | ----------- | ----------- | ----------- |
+| --database       | hoodie.datasource.hive_sync.database  |  default   | Name of the target database in Hive       |
+| --table   | hoodie.datasource.hive_sync.table |       | Name of the target table in Hive. Inferred from the table name in Hudi table config if not specified.        |
```

Comment:
nit: add `(N/A)` to the `Default Value` column for the configs without a default value.